AWSPinpoint/AWSPinpointNotificationManager.m

```diff
@@ -272,6 +329,9 @@ - (AWSPinpointPushActionType) pushActionTypeOfApplicationState:(UIApplicationSta
         case UIApplicationStateInactive:
             pushActionType = AWSPinpointPushActionTypeOpened;
             break;
+        default:
+            pushActionType = AWSPinpointPushActionTypeUnknown;
```

Comment:
pushActionType can be initialized to `AWSPinpointPushActionTypeUnknown`, and  we can remove this line.

Reply:
Same as above, will update